src/Microsoft.AspNetCore.Mvc.Core/Internal/EmptyArray.cs

```diff
@@ -1,19 +1,12 @@
 ﻿// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
-#if !NET451
 using System;
-#endif
 
 namespace Microsoft.AspNetCore.Mvc.Internal
 {
     public static class EmptyArray<TElement>
```

Comment:
Should just remove this class and update places it's used. Class existed only to work around the .NET 4.5's lack of `Array.Empty<T>()`.